### PR TITLE
Revised Grafana password information in monitoring/README

### PR DESCRIPTION
**Pull request by davidbass13**

### CODE REVIEW

**Review by BryanEllington — COMMENTED** [review]
I think it's important to point out that the environment variable can be set via user.env files (either $USER_DIR/user.env or (preferably for this value) $USER_DIR/monitoring/user.env.

File: monitoring/README.md [hunk 1]
```diff
@@ -86,14 +86,20 @@ are present in `USER_DIR`:
...
+- Helm chart parameters: These parameters are organized into a hierarchical structure and contain mixed case characters (for example, `persistentVolume:storageClass`). These parameters are specified in `.yaml` configuration files.
+
 ### user.env
 
-The `monitoring/user.env` file contains environment variable flags that customize
-the components that are deployed or to alter some script behavior (such as to
+The `monitoring/user.env` file contains environment variables that customize
+the components that are deployed or alter some script behavior (such as to
 enable debug output). All values in `user.env` files are exported as environment
 variables available to the scripts. A `#` as the first character in a line
```
Comment (gsmith-sas) [inline monitoring/README.md:98]: The sentence related to comments should be sometnhing like: _"Any line with a '#' as the first character will be treated as a comment and ignored."_

File: monitoring/README.md [hunk 2]
```diff
@@ -193,10 +199,14 @@ applications are available at these locations by default:
 * Prometheus - Port 31090 `http://master-node.yourcluster.example.com:31090`
 * AlertManager - Port 31091 `http://master-node.yourcluster.example.com:31091`
 
-The default Grafana admin user is `admin`. Unless otherwise specified,
-the default password is randomly generated. The value is logged only
-during the initial deployment of the monitoring components and not on
-an upgrade.
+The default Grafana admin user is `admin`. Unless you used the `GRAFANA_ADMIN_PASSWORD` environment variable (either in the `user.env` file or on the command line) to specify a default password during deployment, the default password is randomly generated and displayed during deployment.
```
Comment (gsmith-sas) [inline monitoring/README.md:202]: I think the word "set" is better than "used" (i.e. _Unless you set the 'GRAFANA_ADMIN_PASSWORD' environrment variable..._)

File: monitoring/README.md [hunk 3]
```diff
@@ -86,14 +86,20 @@ are present in `USER_DIR`:
 * `monitoring/user-values-prom-operator.yaml`
 * `monitoring/user-values-pushgateway.yaml`
 
+You can modify two types of values to customize your deployment:
+- Environment variables: These variables contain uppercase characters and underscores (for example, `LOG_COLOR_ENABLE=`), and can be specified in the `user.env` file or on the command line. Specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments.
```
Comment (gsmith-sas) [inline monitoring/README.md:90]: Environment variables do NOT have to use uppercase and underscores...they often do but that is not a requirement.  I do NOT know if every env variable we support follows that convention.  I'm not sure Bryan meant for that to get into the doc...thought it was just internal guidance to help differentiate between env vars and Helm chart parameters.  If it goes in the doc, we need to weaken the statement (perhaps: _These variables often (or generally?) contain uppercase and underscores_)

File: monitoring/README.md [hunk 4]
```diff
@@ -86,14 +86,20 @@ are present in `USER_DIR`:
...
+- Helm chart parameters: These parameters are organized into a hierarchical structure and contain mixed case characters (for example, `persistentVolume:storageClass`). These parameters are specified in `.yaml` configuration files.
```
Comment (gsmith-sas) [inline monitoring/README.md:91]: See comment above about Environment Variables.

File: monitoring/README.md (hunk 3 above)
Comment (davidbass13) [reply monitoring/README.md:90]: Removed information about case and characters in variables

File: monitoring/README.md (hunk 4 above)
Comment (davidbass13) [reply monitoring/README.md:91]: Removed information about case and characters in variables

File: monitoring/README.md (hunk 1 above)
Comment (davidbass13) [reply monitoring/README.md:98]: Changed to "Any line whose first character is `#` is treated as a comment and ignored."

File: monitoring/README.md (hunk 2 above)
Comment (davidbass13) [reply monitoring/README.md:202]: Changed to "set"

File: monitoring/README.md [hunk 5]
```diff
@@ -193,10 +199,14 @@ applications are available at these locations by default:
...
-The default Grafana admin user is `admin`. Unless otherwise specified,
-the default password is randomly generated. The value is logged only
-during the initial deployment of the monitoring components and not on
-an upgrade.
+The default Grafana admin user is `admin`. Unless you used the `GRAFANA_ADMIN_PASSWORD` environment variable (either in the `user.env` file or on the command line) to specify a default password during deployment, the default password is randomly generated and displayed during deployment. 
+
+If you want to change the password, issue this command:
+```bash
+kubectl exec -n <monitoring_namespace> <grafana_pod> -c grafana -- bin/grafana-cli admin reset-admin-password myNewPassword
+```
```
Comment (davidbass13) [inline monitoring/README.md:206]: @BryanEllington, is this the correct command to change the Grafana password?

Comment (BryanEllington) [reply monitoring/README.md:206]: Yes, that is correct

**Pull request merged**